jwql/website/apps/jwql/data_containers.py

```diff
@@ -871,9 +871,26 @@ def get_proposal_info(filepaths):
     return proposal_info
 
 
-def get_thumbnails_all_instruments(instruments):
+def get_thumbnails_all_instruments(instruments, apertures, filters, observing_modes, effexptm_min, effexptm_max, anomalies):
```

Comment:
This could be pushed off to a future PR, but we might consider storing all of these parameters in a dictionary (e.g. ``parameters['instrument`]``, ``parameters['aperture']``, etc.) and pass the dictionary into this function.  That would likely make it easier to add parameters in the future if that is something we want to do.

Reply:
I'll add that in!